Rick-and-Morty/Rick And Morty/ViewModels/CharactersViewModel.swift

```diff
@@ -0,0 +1,48 @@
+import Foundation
+
+final class CharactersViewModel: ObservableObject {
+    let characterType: CharacterType
+    
+    @Published var characters: [Character] = []
+    var title: String = ""
+    var imagePosition: CharacterImagePosition = .right
+    
+    init(characterType: CharacterType) {
+        self.characterType = characterType
+        self.title = getTitle()
+        self.imagePosition = getImagePosition()
+        
+        fetchCharacters()
+    }
+    
+    private func fetchCharacters() {
+        if characterType == .rick {
+            characters = ricks
+        } else {
+            characters = morties
+        }
```

Comment:
The function would be more testable with return value `private func fetchCharacters() -> [Character]`. 

Also for this length of function you could use 
```
characters = characterType == .rick ? ricks : morties
```